odl/tomo/backends/astra_setup.py

```diff
@@ -74,6 +74,70 @@
            'astra_parallel_3d_geom_to_vec')
 
 
+# Dictionary specifying which ASTRA versions support which feature. The dict
+# values can be given as exact version '==1.7' or '1.7', as range '1.7-1.7.4',
+# as inequality '>=1.7' or as comma-separated composition of such (the items
+# will be OR-ed together), e.g. '<=1.7, >=1.7.2'.
+ASTRA_FEATURES = {'anisotropic_voxels_2d': None,
+                  'anisotropic_voxels_3d': '>=1.8',
+                  }
+
+
+def astra_supports(feature):
```

Comment:
Do we truly need this much code for this? Isn't there some built In feature? If so, use it.  Else move the generic parts of this to the util library.